.eslintrc.js

```diff
@@ -15,16 +15,53 @@
  */
 module.exports = {
   'root': true,
+  'env': {
+    'node': true,
+  },
+  'plugins': ['header'],
+  'rules': {
+    'header/header': [
+      2,
+      'block',
+      [
+        '*',
+        {
+          pattern:
+            ' * (© Copyright IBM Corporation 20\\d\\d(?:, 20\\d\\d)?\\. All Rights Reserved\\.|\\(C\\) Copyright IBM Corp\\. 20\\d\\d\\.)',
+          template: ' * © Copyright IBM Corporation 2021. All Rights Reserved.',
+        },
+        ' *',
+        ' * Licensed under the Apache License, Version 2.0 (the "License");',
+        ' * you may not use this file except in compliance with the License.',
+        ' * You may obtain a copy of the License at',
+        ' *',
+        ' *      http://www.apache.org/licenses/LICENSE-2.0',
+        ' *',
+        ' * Unless required by applicable law or agreed to in writing, software',
+        ' * distributed under the License is distributed on an "AS IS" BASIS,',
+        ' * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.',
+        ' * See the License for the specific language governing permissions and',
+        ' * limitations under the License.',
+        ' ',
+      ],
+    ],
+    'max-len': ['error', { 'code': 100 }],
+  },
   'overrides': [
     {
-      'env': {
-        'node': true,
-      },
-      'files': ['**/*.js', '**/*.jsx'],
+      'files': ['**/*.js'],
       'plugins': ['node', 'jest', 'prettier'],
-      'extends': ['airbnb-base', 'plugin:jest/recommended', 'plugin:jest/style', 'prettier'],
+      'extends': [
+        'airbnb-base',
+        'plugin:import/recommended',
+        'plugin:jest/recommended',
+        'plugin:jest/style',
+        'prettier',
+      ],
       'rules': {
         'camelcase': 'off',
+        'no-console': 'off',
+        'strict': 0,
```

Comment:
What does `'strict': 0` mean? An explanation, or the string value of the enum would help.

Reply:
This rule requires or disallows strict mode directives.
Specifically `'strict': 0` turns this check off.
```
  17:1  error  'use strict' is unnecessary inside of modules  strict
```

More details about the rule: https://eslint.org/docs/rules/strict
> In ECMAScript modules, which always have strict mode semantics, the directives are unnecessary.

This project is an ECMAScript module, so no need for strict mode definition.
I can remove this setting, but then I will also remove `'use strict';` literals from the test files.
Actually, not every test file has `'use strict';` specified in the test files. So I think it is better if it is removed.

Reply:
Changed in 159a9ab